docs/docs/apis/actions/introduction.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Actions
+---
+
+This page describes the options you have when writing ZITADEL actions scripts.
```

Comment:
@adlerhurst @hifabienne what I'm completely missing is a link to the actions repository for examples. Could you include that in this file, so that people know where to look for our "marketplace"? I would suggest to put it even in an admonitions/box.

Reply:
I added a sentence in the latest commit